src/main/java/com/epam/test_generator/controllers/StepSuggestionController.java

```diff
@@ -44,6 +44,54 @@
         return new ResponseEntity<>(stepSuggestionService.getStepsSuggestions(), HttpStatus.OK);
     }
 
+    @ApiOperation(value = "Get page of step suggestions", nickname = "getStepsSuggestions")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK", response = StepSuggestionDTO.class,
+                    responseContainer = "List")
+    })
+    @Secured({"ROLE_ADMIN", "ROLE_TEST_ENGINEER", "ROLE_TEST_LEAD", "ROLE_GUEST"})
+    @RequestMapping(value = "/stepSuggestions/page/{pageNumber}/size/{pageSize}", method = RequestMethod.GET,
+            produces = "application/json")
+    @ApiImplicitParam(name = "Authorization", value = "add here your token", paramType = "header", dataType = "string", required = true)
```

Comment:
And apply TAI codestyle

Reply:
Currently I have a problem with applying TAI codestyle, will fix it sooner